gemutils/include/gem/utils/db/GEMDatabaseUtils.h

```diff
@@ -1,7 +1,7 @@
 #ifndef GEM_UTILS_DB_GEMDATABASEUTILS_H
 #define GEM_UTILS_DB_GEMDATABASEUTILS_H
 
-#include "Python.h"
+#include "python2.7/Python.h"
```

Comment:
Revert, i.e., `rebase` and exclude the commit that added this.
If you're having a problem building, please use the mattermost `CMS GEM DAQ/Online Software` channel, or open an issue